### DISCUSSION TITLE
Discord and GitHub connection problem

### DISCUSSION THREAD

**Discussion by AkashSingh0021** [community/community#143246] [Other Feature Feedback, Questions, & Ideas]
When I try to connect my GitHub account to discord it shows can't connect to it error ![Screenshot_2024-11-01-09-49-39-53_e4424258c8b8649f6e67d283a50a2cbc.jpg](https://github.com/user-attachments/assets/848e21e1-0242-4338-9ac9-c6779334f6e0)

**Comment by SyedAliJunaidSherazi** [comment]
Hi there! 

I understand you’re experiencing an issue with connecting your GitHub account to Discord. Here are a few steps that could help resolve this:

1. **Clear Cache and Cookies**: Sometimes, cached data can cause login issues. Try clearing your browser’s cache and cookies, then restart your browser and try connecting again.

2. **Use an Incognito Window**: Open an incognito or private browsing window and attempt the connection there. This can help bypass certain browser extensions or settings that might interfere.

3. **Disable Browser Extensions**: Some extensions, especially privacy-related ones, can block connections. Temporarily disable them and see if the connection succeeds.

4. **Check Permissions on GitHub and Discord**: Ensure that GitHub is authorized to connect with Discord. Go to GitHub’s settings, find “Authorized OAuth Apps,” and ensure that Discord is listed. If not, add it manually.

5. **Try a Different Browser or Device**: If the problem persists, try connecting on a different browser or device to rule out any local configuration issues.

6. **Reach Out to Support**: If none of these steps work, it could be a server-side issue with Discord or GitHub. Consider reaching out to their support for additional help.

I hope this helps! **If my solution resolves your problem, kindly mark this as the accepted answer and do follow me on my GitHub account for more updates on open-source contributions and projects.** 😊

**Comment by Akshatjalan** [comment]
If you’re having trouble connecting your GitHub account to Discord, it could be due to several common issues. Here are a few troubleshooting steps to help resolve it:

1.	Check Discord Permissions:
- Ensure you’re using the latest version of Discord, as older versions may have connectivity issues.
- Make sure you have the necessary permissions in the server where you’re trying to link GitHub (if it’s for a specific server integration).
2.	Verify GitHub and Discord Account Status:
- Ensure both accounts are in good standing and not restricted.
- Make sure you’re logged into the correct GitHub account in your browser when attempting to link it to Discord.
3.	Clear Browser Cache:
- Sometimes cached data can cause connectivity problems. Clear your browser’s cache and cookies, then try reconnecting.
4.	Try a Different Browser or Incognito Mode:
- Occasionally, extensions or settings in the browser can interfere. Try connecting through a different browser or an incognito/private browsing window.
5.	Reconnect OAuth Permissions:
- Go to your GitHub authorized applications and remove Discord if it’s already listed. Then, try reconnecting.
6.	Discord Settings:
- In Discord, go to User Settings > Connections and select GitHub. If you see a pending connection, disconnect and try to re-establish it.

If none of these work, consider reinstalling Discord or contacting Discord support for further assistance.
